client/blocks/app-banner/style.scss

```diff
@@ -0,0 +1,64 @@
+.card.app-banner {
+	position: fixed;
+	bottom: 0;
+	width: 100%;
+	background-color: $blue-light;
+	color: $blue-dark;
+	/* needed for banner to appear in front of notifications panel */
+	z-index: 180;
+
+	background-repeat: no-repeat;
+	background-size: 270px;
```

Comment:
Added a comment in 62a1e26b6537975364533ece7dea5ec6898da675, not sure if that helps.